frontend_app_gallery/test/component/MainPreview.spec.js

```diff
@@ -13,7 +13,12 @@ describe('<MainPreview />', () => {
     handleClickShowImageRaw
   }
 
-  const wrapper = shallow(<MainPreview {...props} t={tradKey => tradKey} />)
+  // INFO - GM - 2020/04/15 - Mock some window functions used by MainPreview
+  window.requestAnimationFrame = (cb) => { cb() }
+  window.getComputedStyle = () => { return {} }
```

Comment:
I think it would fit best in test/setup.js
see  window.matchMedia for example

Also, since requestAnimationFrame is supposed not to answer immediately, wrap the cb() in a setTimeout of, I don't know, 10ms should be fine. It'll reflect better what is happening in browser